scripts/auth-open.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
```

Comment:
The purpose of this script is entirely unclear. If anything it should live in test_util, not a brand new top-level directory